src/client/discord.cpp

```diff
@@ -0,0 +1,79 @@
+#include "client\discord.h"
+#include <ctime>
+#include "discord_rpc.h"
+#include <string>
+
+std::unique_ptr<Discord> Discord::s_pDiscord;
+
+Discord::Discord()
+{	
+	m_data.startTimestamp = time(0);
+}
+
+Discord::~Discord()
+{
+	Discord_Shutdown();
+}
+
+void Discord::create()
+{
+	s_pDiscord = std::make_unique<Discord>();
```

Comment:
std::make_unique is C++17 and greater. We are on c++11

Reply:
C++14 but okay I remove this